src/tests/multihost/alltests/test_sssctl_local.py

```diff
@@ -80,3 +81,49 @@ def test_0002_bz1599207(self, multihost,
                                                   raiseonerr=False)
             assert 'Cache entry creation date' in \
                    cmd.stdout_text and cmd.returncode == 0
+
+    @pytest.mark.tier1
+    def test_0003_bz1661182(self, multihost,
+                            backupsssdconf):
+        """
+        :title: sss_cache prints spurious error messages
+        when invoked from shadow-utils on package install
+        :id: 8f2868d2-1ece-11ec-ac6d-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1661182
+        :steps:
+          1. Configure sssd without any domain
+          2. Restart sssd (sssd should not be
+           running after this)
+          3. Modify existing local user usermod -a -G wheel user1
+          4. This message
+           [sss_cache] [confdb_get_domains] (0x0010):
+           No domains configured, fatal error!
+           must not appear in console
+        :expectedresults:
+          1. Should succeed
+          2. Should succeed
+          3. Should succeed
+          4. Should succeed
+        """
+        tools = sssdTools(multihost.client[0])
+        ldap_params = {'enable_files_domain': 'false'}
+        tools.sssd_conf('sssd', ldap_params)
+        with pytest.raises(SSSDException):
+            multihost.client[0].service_sssd('restart')
```

Comment:
Isn't it better to stop sssd and clear cache instead of failing a restart?

Reply:
Its part of bug verify step.:

https://bugzilla.redhat.com/show_bug.cgi?id=1661182#c7